selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/NewWorkspace.java

```diff
@@ -76,6 +76,9 @@ public NewWorkspace(SeleniumWebDriver seleniumWebDriver, ActionsFactory actionsF
     String WORKSPACE_CREATED_DIALOG = "//md-dialog/che-popup[@title='Workspace Is Created']";
     String EDIT_WORKSPACE_DIALOG_BUTTON = "//che-button-primary//span[text()='Edit']";
     String OPEN_IN_IDE_DIALOG_BUTTON = "//che-button-default//span[text()='Open In IDE']";
+
+    String ORGANIZATIONS_LIST = "namespace-selector";
```

Comment:
Please, add `_ID` suffix to make it clear that this is _ID_ value of organization list.